tests/unit/test_credentials.py

```diff
@@ -746,6 +746,46 @@ def test_account_id_with_invalid_arn(self):
         self.assertEqual(response, expected_response)
         self.assertEqual(response['account_id'], None)
 
+    def test_feature_ids_property(self):
```

Comment:
I don't see too much value in both of the `test_feature_ids_property` tests. It looks like they're just checking that `@property` works as expected?